crates/pathfinder/src/bin/pathfinder.rs

```diff
@@ -35,12 +35,13 @@ async fn main() -> anyhow::Result<()> {
         .await
         .context("Determining Ethereum chain")?;
 
-    let database_path = match network_chain {
+    let database_path = config.data_directory.join(match network_chain {
         ethereum::Chain::Mainnet => "mainnet.sqlite",
         ethereum::Chain::Goerli => "goerli.sqlite",
-    };
+    });
+    let storage = Storage::migrate(database_path.clone()).unwrap();
+    info!(location=?database_path, "Database loaded.");
```

Comment:
```suggestion
    info!(location=?database_path, "Database migrated");
```

Having loaded a database just seems a weird choice of words, because no we are not intentionally reading up the whole file to make sure it's in the page cache for example.

Reply:
I originally decided against `"migrated"` because it may imply that we are migrating every time, even if there were no changes.